Casks/v/vcam.rb

```diff
@@ -0,0 +1,44 @@
+cask "vcam" do
+  version "2.0.133"
+  sha256 "e88aa8ae051bbb82deb112657bd1ece69fbc90c7f06dcfef7e26809103f6fe99"
+
+  url "https://downloads.vcam.ai/mac/VCam.ai_#{version}.pkg"
+  name "VCam"
+  desc "Webcam background tool"
+  homepage "https://vcam.ai/"
+
+  livecheck do
+    url "https://downloads.vcam.ai/mac/updates.txt"
+    regex(/ProductVersion = (\d+(?:\.\d+)+)/i)
+  end
+
+  depends_on macos: ">= :catalina"
+
+  pkg "VCam.ai_#{version}.pkg"
+
+  preflight do
+    retries ||= 3
+    ohai "Attempting to close VCam.app to avoid unwanted user intervention" if retries >= 3
+    return unless system_command "/usr/bin/pkill", args: ["-f", "/Applications/VCam.ai/VCam.app"]
+  rescue RuntimeError
+    sleep 1
+    retry unless (retries -= 1).zero?
+    opoo "Unable to forcibly close VCam.app"
+  end
```

Comment:
I assume that the `pkg` installer opens the application?
We should probably document this specifically with a comment if that is the case.

Reply:
Oops I also used the wrong snippet
```suggestion
  postflight do
    # Description: Ensure console variant of postinstall is non-interactive.
    # This is because `open /Applications/VCam.ai/VCam.app` is called from the
    # postinstall script of the package and we don't want any user intervention there.
    retries ||= 3
    ohai "The VCam.ai package postinstall script launches the VCam app" if retries >= 3
    ohai "Attempting to close VCam.app to avoid unwanted user intervention" if retries >= 3
    return unless system_command "/usr/bin/pkill", args: ["-f", "/Applications/VCam.ai/VCam.app"]
  rescue RuntimeError
    sleep 1
    retry unless (retries -= 1).zero?
    opoo "Unable to forcibly close VCam.app"
  end
```